src/routes/Router.tsx

```diff
@@ -32,14 +38,32 @@ const Router = () => {
         <Route path="/legal/termsAndConditions" element={<Legal document="termsAndConditions" />} />
         <Route path="/legal/privacyPolicy" element={<Legal document="privacyPolicy" />} />
         <Route path="/legal/cookiePolicy" element={<Legal document="cookiePolicy" />} />
+        {renderLegacyRoute(legacyCreateBoard)}
         <Route
-          path="/new"
+          path="/boards"
           element={
             <RequireAuthentication>
-              <NewBoard />
+              <Boards />
             </RequireAuthentication>
           }
-        />
+        >
+          <Route index element={<Navigate to="templates" />} />
+          <Route path="templates" element={<Templates />}>
+            {/* TODO extract settings routes to avoid repetition */}
+            <Route path="settings" element={<SettingsDialog enabledMenuItems={{appearance: true, feedback: feedbackEnabled, profile: true}} />}>
+              <Route path="appearance" element={<Appearance />} />
+              <Route path="feedback" element={<Feedback />} />
+              <Route path="profile" element={<ProfileSettings />} />
+            </Route>
+          </Route>
+          <Route path="sessions" element={<Sessions />}>
+            <Route path="settings" element={<SettingsDialog enabledMenuItems={{appearance: true, feedback: feedbackEnabled, profile: true}} />}>
+              <Route path="appearance" element={<Appearance />} />
+              <Route path="feedback" element={<Feedback />} />
+              <Route path="profile" element={<ProfileSettings />} />
+            </Route>
+          </Route>
+        </Route>
```

Comment:
These routes should be `/templates` and `/sessions` instead of `/board/templates` and `/board/sessions`

Reply:
it's this way on purpose because I wanted to take advantage of Outlets, so we have the same outer components (i.e. title, searchbar and so on) while the main content (Templates, Sessions) get replaced based on the route.

this is definitely not the only way to approach this, but i settled like half a year ago so changing this would have some serious consequences on how the layout is implemented.

I can take a look if I can get it to work with removing the /board/ but if not I would leave it as is.